x-pack/plugins/security_solution/cypress/tasks/rule_details.ts

```diff
@@ -89,13 +125,14 @@ export const goToAlertsTab = () => {
 };
 
 export const goToExceptionsTab = () => {
+  cy.get(EXCEPTIONS_TAB).should('exist');
   cy.get(EXCEPTIONS_TAB).click();
 };
 
 export const editException = () => {
-  cy.get(EXCEPTION_ITEM_ACTIONS_BUTTON).click({ force: true });
+  cy.get(EXCEPTION_ITEM_ACTIONS_BUTTON).eq(0).click({ force: true });
```

Comment:
Do we have more than one element now with the same locator?

Reply:
We do, there's a test with multiple exception items in the items table. Updated to take an index.